keyboards/crkbd/keymaps/via/keymap.c

```diff
@@ -70,8 +70,10 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 };
 
 #ifdef OLED_DRIVER_ENABLE
+#include <stdio.h>
```

Comment:
Same question as with for the `default` keymap.

Reply:
Same answer